File: src/main/java/com/gmail/borlandlp/minigamesdtools/activepoints/behaviors/BehaviorCreatorHub.java
package com.gmail.borlandlp.minigamesdtools.activepoints.behaviors;

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider;
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub;

public class BehaviorCreatorHub extends CreatorHub {
    public Behavior createBehavior(String ID, AbstractDataProvider dataProvider) throws Exception {
        return (Behavior) this.create(ID, dataProvider);
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/activepoints/behaviors/BehaviorCreatorHub.kt
package com.gmail.borlandlp.minigamesdtools.activepoints.behaviors

import com.gmail.borlandlp.minigamesdtools.creator.AbstractDataProvider
import com.gmail.borlandlp.minigamesdtools.creator.CreatorHub

class BehaviorCreatorHub : CreatorHub() {
    @Throws(Exception::class)
    override fun create(itemID: String, dataProvider: AbstractDataProvider): Behavior {
        return super.create(itemID, dataProvider) as Behavior
    }
}